starters/angular-apollo-tailwind/src/styles.scss

```diff
@@ -1,5 +1,5 @@
 /* You can add global styles to this file, and also import other style files */
 
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import "tailwindcss/base";
+@import "tailwindcss/components";
+@import "tailwindcss/utilities";
```

Comment:
I don't think this is the right fix here. We may need to install the tailwind plugin.

Reply:
Corrected.